kornia/losses/focal.py

```diff
@@ -122,3 +122,93 @@ def forward(  # type: ignore
             input: torch.Tensor,
             target: torch.Tensor) -> torch.Tensor:
         return focal_loss(input, target, self.alpha, self.gamma, self.reduction, self.eps)
+
+
+def binary_focal_loss_with_logits(
+        inp: torch.Tensor,
+        target: torch.Tensor,
+        alpha: float = .25,
+        gamma: float = 2.0,
+        reduction: str = 'none',
+        eps: float = 1e-8) -> torch.Tensor:
+    r"""Function that computes Binary Focal loss.
+
+    """
+    if not torch.is_tensor(inp):
```

Comment:
@xen0f0n you need to modify this line by: `if not isinstance(inp, torch.Tensor):` since `torch.is_tensor(inp)` is not supported by JIT